frontend/packages/dev-console/console-extensions.json

```diff
@@ -435,5 +435,54 @@
       "id": "resources",
       "perspective": "dev"
     }
+  },
+  {
+    "type": "console.topology/component/factory",
+    "properties": {
+      "getFactory": {
+        "$codeRef": "topology.getDevConsoleComponentFactory"
+      }
+    }
+  },
+  {
+    "type": "console.topology/data/factory",
+    "properties": {
+      "id": "devconsole-bindable-topology-model-factory",
+      "priority": 200,
+      "resources": {
+        "bindables": {
+          "model": { "kind": "IntegrationPlatform", "group": "camel.apache.org", "version": "v1" },
+          "opts": {
+            "isList": true,
+            "optional": true
+          }
+        }
+      },
```

Comment:
This is done only to test resources as bindable so one we have is Kafka connection resouce conributes with Rhoas operator and i add `IntegrationPlatform` once CR for `IntegrationPlatform` is created pls add a label 

```
app.kubernetes.io/component: external-service
```

the actual contribution of resources will happen with https://issues.redhat.com/browse/ODC-6160